libs/gltfio/src/AssetLoader.cpp

```diff
@@ -1159,7 +1155,6 @@ void FAssetLoader::addTextureBinding(MaterialInstance* materialInstance, const c
     }
 
     mResult->mTextureSlots.push_back({
-        .texture = srcTexture,
```

Comment:
Is removal of this line intentional?

Reply:
No it wasn't, I already uploaded a revert of this